5-20/5-20.md

```diff
@@ -541,42 +538,46 @@ func updateUser(u *user) (*updateStats, error) {
 Updated user record for ID 101
 ```
 
-#### Constant
+#### 상수
 
 Constants are not variables. Constants have a parallel type system all to
 themselves. The minimum precision for constant is 256 bit. They are considered to
 be mathematically exact. Constants only exist at compile time.
 
-##### Declare and initialize
+상수는 변수가 아니며 상수를 위한 parallel type system[재방문] 이 있다.
+상수의 최소 정밀도(minimum precision)는 265 bit 이며, 이 정도의 정밀도는 수학적으로 정확하다고 간주한다.
+상수는 컴파일을 하는 동안만 존재한다.[재방문: 설명필요]
```

Comment:
```suggestion
상수는 주소를 가지고 있지 않기 때문에 컴파일 시간에만 존재한다. [재방문: 단순히 컴파일 시간에만 존재하는 이유가, 주소를 갖고 있지 않는 하나의 이유때문인지 확인 필요]
```

Reply:
다음은 참고한 설명과 링크 입니다.

`In Go, constants do not have addresses because they only exist as constants at compile time. There is more detailed information here: Constants - The Go Blog

But the basic idea is that a constant does not have an exact type in the sense that Go variables do. At compile time, their type is inferred from context and when one is actually used and assigned to a variable, a corresponding Go type is assigned, and a memory location according to whatever rules are involved with the variable creation. Only at that point is there a memory address.

Variables (generally) have addresses. Constants are not variables, so they don’t have addresses, although you can convert them to a variable and take the address of that variable.`

주소(https://www.quora.com/Can-you-get-the-address-of-a-constant-in-Golang)

Reply:
감사합니다. 풋노트로 해당하는 부분을 추가하여 넣도록 하겠습니다. 

1. 상수는 변수와는 달리 정확한 타입이 없다. 
2. 컴파일할때에 상황(=문맥)에 따라 타입을 유추하며, 실제로 쓰이게 될 때에 적합한 타입과 메모리를 할당한다. 이 순간에 메모리 주소가 있다.